topics/request.md

```diff
@@ -1,316 +1,116 @@
-[//]: # (title: Request)
+[//]: # (title: Send a request)
 
-<include src="lib.md" include-id="outdated_warning"/>
+After adding the necessary [dependencies](client.md#add-dependencies) and [creating the client](client.md#create-client), you can send HTTP requests. You can configure the following request parameters:
+* Specify an HTTP method, such as `GET`, `POST`, `PUT`, and so on.
+* Add headers and cookies.
+* Set the body of a request, for example, a plain text, a data object, form parameters, and so on.
 
-## Making request
-
-After client configuration we're ready to perform our first request.
-Most of the simple requests are made with pattern
-
-```kotlin
-val response = client.'http-method'<'ResponseType'>("url-string")
-```
-
-or even simpler form(due to kotlin generic type inference):
-
-```kotlin
-val response: ResponseType = client.'http-method'("url-string")
-```
-
-For example to perform a `GET` request fully reading a `String`:
-
-```kotlin
-val htmlContent = client.get<String>("https://en.wikipedia.org/wiki/Main_Page")
-// same as
-val content: String = client.get("https://en.wikipedia.org/wiki/Main_Page")
-```
-
-And in the case you are interested in the raw bits, you can read a `ByteArray`:
-
-```kotlin
-val channel: ByteArray = client.get("https://en.wikipedia.org/wiki/Main_Page")
-```
-
-Or get full [HttpResponse](https://api.ktor.io/%ktor_version%/io.ktor.client.statement/-http-response/index.html):
+The main way for making HTTP requests is the [request](https://api.ktor.io/%ktor_version%/io.ktor.client.request/request.html) function that takes a URL as a parameter. Inside this function, you can configure various request parameters: specify an HTTP method, add headers, specify the request body and content type, and so on. These parameters are exposed by the [HttpRequestBuilder](https://api.ktor.io/%ktor_version%/io.ktor.client.request/-http-request-builder/index.html) class.
```

Comment:
The first sentence feels like a repeat of the above, which leads me to think that the above section can be removed entirely.

Reply:
agree, rephrased the intro